six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -124,6 +124,28 @@ Data* readNITF(const std::string& pathname,
     return reinterpret_cast<Data*>(reader.interleaved(region, 0));
 }
 
+
+void crop_SICD(const std::string& inPathname,
+               const std::vector<std::string>& schemaPaths,
+               const types::RowCol<size_t>& aoiOffset,
+               const types::RowCol<size_t>& aoiDims,
+               const std::string& outPathname);
+              
+void crop_SICD(const std::string& inPathname,
```

Comment:
`crop_SICD()` --> `cropSICD()`

Reply:
I called it crop_SICD because there is already a function in CropUtils.h with the name cropSICD and the same arguments (the one I am wrapping).  Is there a better way to get around this?

Reply:
@JonathanMeans can you show him how to do this?  There's a way with (I think it's) `%ignore` that you can tell Swig to ignore certain overloadings but still let some through... that way we don't need the underscore in these.

Reply:
Try adding `%rename(_cropSICDImpl) six::sicd::cropSICD;`

Reply:
I tried renaming and ignoring, but it didn't work.  I think the issue is that the CropUtils.h is being included in the generated six_sicd.swigwrap_2.cxx file and I am trying to define a new c++ function with the same signature as one in CropUtils.h.  The error is happening when I gcc is compiling six_sicd.swigwrap_2.cxx:
```
six/modules/python/six.sicd/source/six_sicd.swigwrap_2.cxx:3566:25: error: call of overloaded ‘cropSICD(const string&, const std::vector<std::basic_string<char> >&, const types::RowCol<long unsigned int>&, const types::RowCol<long unsigned int>&, const string&)’ is ambiguous
     aoiDims, outPathname);
```
The only reason I have this function is call six::XMLControlFactory::getInstance().addCreator(),